geometry_msgs/Pose2D pose
float64 sail_angle
float64 rudder_angle